### ISSUE TITLE
I got the following error and I even reinstall Flagrum but still i get the this error...i cant do anything...unsub and sub the mods again fix the error but installing rinual's mod menu bring this error again

### ISSUE THREAD

**Issue by sharknizer** [Kizari/Flagrum#91]
A Tasks exception(s) were not observed either by waiting on the task or accessing its exception properly. As a result, the unobserved exception was rethrown by the finalizer thread. (Exception of type 'Flagrum.Web.Utilities.UserFriendlyException' was thrown.)

The application will now close.

**Comment by Kizari** [comment]
Potential duplicate of #82 (exact same exception message)

Please try the workarounds discussed in there and reopen this issue with the following details if none of the solutions there fix your problem:
* The steps you took leading up to the bug occurring
* What you expected to occur instead of the bug
* Attach your crash report found on your PC at %LOCALAPPDATA%\Flagrum\crash.txt

**Comment by sharknizer** [comment]
so this error started pooping when i try to add clothing for FFXV comrades and when i open the game i saw no mods that i installed so i opened Flagrum and it hit me with this error...the mods i installed manually were downloaded from the skymod website and the owner of the mods has removed its mods from steams and also their steam profile.
After that i removed those mods and the error keeps pooping when flagrum is opened...i try to reinstalled flagrum but the error stayed...i even unsub and sub the mods that i have in the game and still nothing...
![Screenshot (2)](https://github.com/Kizari/Flagrum/assets/145669490/26a33c14-c56c-40ea-ac0d-e3c022941c7c)

**Comment by sharknizer** [comment]
> Potential duplicate of #82 (exact same exception message)
> 
> Please try the workarounds discussed in there and reopen this issue with the following details if none of the solutions there fix your problem:
> 
> * The steps you took leading up to the bug occurring
> * What you expected to occur instead of the bug
> * Attach your crash report found on your PC at %LOCALAPPDATA%\Flagrum\crash.txt

so this error started pooping when i try to add clothing for FFXV comrades and when i open the game i saw no mods that i installed so i opened Flagrum and it hit me with this error...the mods i installed manually were downloaded from the skymod website and the owner of the mods has removed its mods from steams and also their steam profile.
After that i removed those mods and the error keeps pooping when flagrum is opened...i try to reinstalled flagrum but the error stayed...i even unsub and sub the mods that i have in the game and still nothing...
![Screenshot (2) (1)](https://github.com/Kizari/Flagrum/assets/145669490/00909ca9-47a9-47ad-9ea4-2fdb00342cd1)

**Comment by Kizari** [comment]
As mentioned in my previous response, your issue is resolved by the steps outlined in #82. Your crash log confirms that it is indeed the same problem. Make sure to scroll down and read the whole discussion if the measures in the first couple of posts didn't help.
